core/src/test/java/io/confluent/rest/CustomInitTest.java

```diff
@@ -143,7 +144,7 @@ public void onError(Throwable t) {
 
     WebSocket ws = Dsl.asyncHttpClient()
         .prepareGet(uri + "/test")
-        .setRequestTimeout(10000)
+        .setRequestTimeout(Duration.ofSeconds(10000))
```

Comment:
this seems a bit odd, should it be `Duration.ofMillis(10000)`? Because 10000 seconds are quite long.